pkg/config/load_config_test.go

```diff
@@ -1,127 +0,0 @@
-package config
-
-import (
-	"bytes"
-	"fmt"
-	"net/http/httptest"
-	"net/url"
-	"os"
-	"strings"
-	"testing"
-
-	"github.com/johannesboyne/gofakes3"
-	"github.com/johannesboyne/gofakes3/backend/s3mem"
-	"github.com/spf13/afero"
-	"github.com/stretchr/testify/assert"
-)
-
-const bucketName = "myBucket"
-const defaultPermissions = 0644
-
-func putFile(backend gofakes3.Backend, path, mime, content string) error {
-	u, err := url.Parse(path)
-	if err != nil {
-		return err
-	}
-	_, err = backend.PutObject(
-		bucketName,
-		strings.TrimPrefix(u.Path, "/"),
-		map[string]string{"Content-Type": mime},
-		bytes.NewBufferString(content),
-		int64(len(content)),
-	)
-
-	return err
-}
-
-func setupTestS3Bucket(t *testing.T) (*url.URL, *s3mem.Backend) {
-	backend := s3mem.New()
-	faker := gofakes3.New(backend)
-
-	srv := httptest.NewServer(faker.Server())
-
-	t.Cleanup(srv.Close)
-
-	assert.NoError(t, backend.CreateBucket(bucketName))
-	u, err := url.Parse(srv.URL)
-	assert.NoError(t, err)
-	return u, backend
-}
-
-func TestLoadRemoteFile(t *testing.T) {
```

Comment:
Moved into `loader_test.go` and rename to `Test_LoadHCLFile` as it tests both local and remote loading